src/category_theory/abelian/basic.lean

```diff
@@ -109,6 +117,26 @@ open category_theory
 namespace category_theory.abelian
 variables {C : Type u} [category.{v} C] [abelian C]
 
+/-- An abelian category has finite biproducts. -/
+def has_finite_biproducts : has_finite_biproducts.{v} C :=
```

Comment:
```suggestion
def has_finite_biproducts : has_finite_biproducts C :=
```
and elsewhere in both files. Now that we don't need the universe annotations, let's get rid of them --- they only scare people! :-)